scripts/examples/nrfconnect_example.sh

```diff
@@ -39,4 +39,9 @@ source "../scripts/activate.sh"
 export GNUARMEMB_TOOLCHAIN_PATH="$PW_PIGWEED_CIPD_INSTALL_DIR"
 env
 
-west build -b "$BOARD" -d "$APP/nrfconnect/build/$BOARD" "$APP/nrfconnect" -- "$@"
+OVERLAY_CONF_FLAGS=()
+if [[ -f "$APP/nrfconnect/boards/$BOARD.conf" ]]; then
+    OVERLAY_CONF_FLAGS+=(-DOVERLAY_CONFIG=boards/"$BOARD".conf)
```

Comment:
This shouldn't be needed. `lighting-app/nrfconnect/CMakeLists.txt` contains the following:
```
 if (EXISTS boards/${BOARD}.conf)
   list(APPEND CONF_FILE boards/${BOARD}.conf)
 endif()
```
The problem is that it expects that the build is triggered from the project's directory. You can just fix it with:
```
if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/boards/${BOARD}.conf)
```
and then the build scripts won't need to be updated.

Reply:
Thanks for the comment, that's a much better fix! I was hoping not to have to do the ugly handling in the scripts. I have updated all the nrfconnect examples' CMakeList.txt to have this fix. The shell app and the pigweed-app examples' CMakeList.txt didn't have this block at all so I added it.

Reply:
Thank you :)